cmd/aws-s3-provisioner.go

```diff
@@ -20,95 +20,108 @@ package main
 import (
 	"errors"
 	"flag"
+	"fmt"
 	"os"
 	"path"
 	"syscall"
 
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/s3"
+
 	"github.com/golang/glog"
-	"github.com/yard-turkey/lib-bucket-provisioner/controller"
+	v1alpha1 "github.com/yard-turkey/lib-bucket-provisioner/pkg/apis/objectbucket.io/v1alpha1"
```

Comment:
nit: import alias need needed since it's the same as the pkg name here and below.